### PR TITLE
Updated Catalog admin

### LINKED ISSUE #200: As maintainer I want the field "is van" of statustype, roltype, eigenschap removed and replaced by a clear presentation of the zaaktype at the top of the page
... so I know the context of the statustype, roltype, eigenschap I am editing.

Similarly, this holds for resultaattype. Field is called "is relevant voor".

**Description**
Describe the user story in more detail.

### PR DESCRIPTION
**Pull request by joeribekker**
Fixes #200 

image

### CODE REVIEW

File: src/openzaak/components/catalogi/openapi.yaml [hunk 1]
```diff
@@ -8186,7 +8186,7 @@ components:
           type: string
           maxLength: 1000
         zaaktype:
-          title: Is van
+          title: Zaaktype
```
Comment (sergei-maertens) [inline src/openzaak/components/catalogi/openapi.yaml]: this shouldn't happen, right?

File: src/openzaak/components/zaken/openapi.yaml [hunk 2]
```diff
@@ -24,13 +24,9 @@ info:
...
     \ RSIN van de Niet-natuurlijk persoon zijnde de organisatie die de zaak heeft\
     \ gecreeerd. Dit moet een geldig RSIN zijn van 9 nummers en voldoen aan https://nl.wikipedia.org/wiki/Burgerservicenummer#11-proef\n\
-    * `zaaktype`: URL-referentie naar het ZAAKTYPE (in de Catalogi API) in de CATALOGUS\
-    \ waar deze voorkomt\n* `vertrouwelijkheidaanduiding`: Aanduiding van de mate\
-    \ waarin het zaakdossier van de ZAAK voor de openbaarheid bestemd is.\n\n**Resources\
-    \ en acties**\n- `zaak`: create, update, destroy\n- `status`: create\n- `zaakobject`:\
-    \ create\n- `zaakinformatieobject`: create\n- `zaakeigenschap`: create\n- `klantcontact`:\
-    \ create\n- `rol`: create\n- `resultaat`: create, update, destroy\n- `zaakbesluit`:\
-    \ create\n\n\n**Handige links**\n\n* [Documentatie](https://zaakgerichtwerken.vng.cloud/standaard)\n\
+    * `zaaktype`: URL-referentie naar het ZAAKTYPE (in de Catalogi API).\n* `vertrouwelijkheidaanduiding`:\
```
Comment (sergei-maertens) [inline src/openzaak/components/zaken/openapi.yaml]: stuk notificaties docs is weg

File: src/openzaak/utils/admin.py [hunk 3]
```diff
@@ -299,3 +299,81 @@ def get_formset(self, request, obj=None, **kwargs):
...
+
+class UUIDAdminMixin:
+    def get_list_display(self, request):
+        list_display = super().get_list_display(request)
+        return tuple(list_display) + ("_get_uuid_display",)
+
+    def _get_uuid_display(self, obj):
+        from django.utils.safestring import mark_safe
+
+        return mark_safe(
```
Comment (sergei-maertens) [inline src/openzaak/utils/admin.py]: better to use `from django.utils.html import format_html` - you interpolate, django escapes what needs to be escaped and you don't have to use `mark_safe`

File: src/openzaak/components/zaken/openapi.yaml (hunk 2 above)
Comment (joeribekker) [reply src/openzaak/components/zaken/openapi.yaml]: Ik zag het maar begreep niet hoe dat door mij kwam dus nam aan dat iemand anders een change had gedaan.

Ik ga er achter aan.

File: src/openzaak/components/catalogi/openapi.yaml (hunk 1 above)
Comment (joeribekker) [reply src/openzaak/components/catalogi/openapi.yaml]: Oh, I thought there weren't any changes. I'll fix.

Comment (joeribekker) [reply src/openzaak/components/catalogi/openapi.yaml]: I'm keeping it this way. We'll solve title in another way and I don't find it important enough.

File: src/openzaak/components/zaken/openapi.yaml (hunk 2 above)
Comment (joeribekker) [reply src/openzaak/components/zaken/openapi.yaml]: This seems to be an issue in the master branch as well. It was just not updated with the schema's.

I'll create an issue for this.

**Pull request merged**